### PR TITLE
Add JUnit-SearchCommandTest

### LINKED ISSUE #57: JUnit-Test SearchCommandTest

### PR DESCRIPTION
**Pull request by SimJJ96**
Fixes #57

### CODE REVIEW

**Review by kwokyto — APPROVED** [review]
LGTM

File: src/main/java/seedu/fridgefriend/command/SearchCommand.java [hunk 1]
```diff
@@ -22,26 +23,39 @@ public SearchCommand(String foodName) throws EmptyDescriptionException {
...
-            message = "You have " + foodName + " in your fridge.";
+        if (indexOfFoodStored >= 0) {
+            message = "You have " + foodName + " stored in "
+                    + Fridge.getFood(indexOfFoodStored).getStorageLocation()
+                    + " of your fridge.";
         } else {
             message = "You do not have " + foodName + " in your fridge.";
         }
-        Ui.printMessage(message);
+        return message;
```
Comment (Leeyp) [inline src/main/java/seedu/fridgefriend/command/SearchCommand.java:47]: Can I understand what is the rationale for the change?

**Review by Leeyp — APPROVED** [review]
LGTM

**Review by Vinci-Hu — APPROVED** [review]
LGTM
